schemas/test/en/test_list_collector_section_summary_items.json

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
We should have a schema where we don't have any related answers, i.e. to show we can have a section summary for a list collector that will be used with repeating sections instead. It doesn't need to be a separate schema, it should be a separate section.

Reply:
I've added extra section to the new schema for this PR, so after a section that showcases related answers there is another one without them.

Reply:
The extra section you added if using a normal list summary. It should still be a section summary with a list but no related. So you need `show_non_item_answers: true`.

Reply:
I've updated the additional section now 👍

Reply:
Just noticed when we have `related_answers` the item label is duplicated and looks odd. It prob needs to be more like a list summary when we don't have related answers. Happy to address in future, probably not needed initially and we need to get this i.
<img width="692" alt="image" src="https://user-images.githubusercontent.com/35296336/207298679-30473071-4f8e-455c-a789-acc85f461abc.png">